src/ui/setup/dynamic/inspector.ts

```diff
@@ -0,0 +1,16 @@
+import { extendStore } from "../store";
```

Comment:
I'm curious why this was refactored up to the shared ui space if it's only used by the inspector. Maybe the real question is why the inspector needs dedicated space in the store if it's the only one using the data. /shrug

Reply:
I kept all the setup logic in one folder, but I'm open to organizing it differently.

> Maybe the real question is why the inspector needs dedicated space in the store if it's the only one using the data

Where else would you put that data?

Reply:
My inclination is to keep local state at the component and move it to redux when it needs to be used broadly across the app. That does add some mental complexity though since we have to then keep track of what is "shared app state" vs "local component state" but can ultimately lead to a smaller app state tree which is easier to grok than "every piece of state used anywhere in the app ever". :)